qprofiler2/src/org/qcmg/qprofiler2/bam/BamSummarizerMT2.java

```diff
@@ -394,21 +380,14 @@ public void run() {
 					}
 					iter.close();
 				}
-				
-			} catch (InterruptedException e) {
+				reader.close();
```

Comment:
you don't need to close if using a try with resources block, and the SamReader implements AutoCloseable